wavesrv/pkg/cmdrunner/cmdrunner.go

```diff
@@ -3560,6 +3563,45 @@ func SessionSetCommand(ctx context.Context, pk *scpacket.FeCommandPacketType) (s
 	return update, nil
 }
 
+func SleepCommand(ctx context.Context, pk *scpacket.FeCommandPacketType) (scbus.UpdatePacket, error) {
```

Comment:
Not sure about this
For my slash command "scripts" it waits until the http call returns before running the next one, there is no update or anything, so for now at least, sleep has to lock the current process. I think this isn't too big a deal because we are already running inside of a goroutine? but either way I didn't end up using this so I can delete it if it seems sketchy